src/annotator/config/test/index-test.js

```diff
@@ -91,6 +91,7 @@ describe('annotator.config.index', function() {
     'branding',
     'openSidebar',
     'requestConfigFromFrame',
+    'configFrameAncestorLevel',
```

Comment:
I think this change is left over from the first iteration of this PR and no longer applicable?